### ISSUE TITLE
Adding pathways for players hostile to the Quarg in the Wanderer storyline

### ISSUE THREAD

**Issue by Amazinite** [endless-sky/endless-sky#2909]
This is essentially related to #1148 from a while back.

Currently if you are hostile with the Quarg, you can not complete the Wanderer campaign as certain missions require you to land on the Quarg ringworld in Hai space. While currently I can consider brushing this aside as the Wanderer storyline has yet to be completed, I think that alternative missions for players hostile to the Quarg should be considered for the future, likely to be written after the storyline is done. Such alternatives would obviously be more difficult though.

To block a player from finishing the Wanderer campaign for stuff they did to the Quarg kind of seems counter to the general idea of having a choose-your-own-adventure game. Yes, being hostile toward the Quarg should close a number of opportunities, but I don't think it's fair to block completion of the Wanderer storyline, especially since the Quarg are not a central part of that storyline and not exactly a faction that the player might expect to need to interact with when starting the campaign. (A player seeing the increase in combat difficulty might even try to plunder Quarg tech to try to make it easier, only to find that this also breaks the campaign.) The unforgiving nature of the Quargs' reputation toward the player essentially leads players looking to finish the story to either to 1) cheat (which most people who play the game likely won't know how to do) or 2) make a new pilot and replay all the way up to that point in the story.

Wanderer missions where the player must interact with the Quarg and possible alternatives:
* "Wanderers: Alpha Surveillance C": Meet Elias Hanover and obtain the device able to track ships using jump drives
  * Alternative: the player must scout out on their own to try and find where the Alphas are hiding, possibly being given the hint that they must not be too far from the Unfettered.
* "Wanderers: Quarg Assistance 1": Rek asks you to meet with the Quarg and talk to them about the Wanderers coming to Korath space
  * Rek, likely knowing that you and the Quarg are hostile to one another, could ask that you instead return to the Wanderers and ask them to send an Ambassador to the Quarg. The Quarg could agree that they will not interfere with the Wanderers, but knowing that you are with the Wanderers they won't send any ships.
* "Wanderers: Kor Efret 3": The Quarg help to defend the Kor Efreti against a Mereti attack
  * The Quarg, seeing that you are trying to assist the Kor Efreti, may possible become temporarily friendly? `" Quarg "` government? Either that or they send less ships to help defend knowing that they will also have to fight the player
* "Wanderers: Sestor: Quarg Help 1": Danforth sends you to try and get some Quarg assistance
  * The player likely has Quarg tech since they are hostile toward the Quarg, so the following missions are made harder by the fact that the Quarg are not helping but might be easy if the player has Quarg tech as assumed.
  * Or the Quarg allow the player to land on the ringworld knowing that humanity is under attack, and the player and the Quarg have a temporary truce to defeat the Sestor invasion (another use of a `" Quarg "` government).

There are obviously more missions where mention is made of the Quarg under the assumption that these missions were done with the Quarg being friendly, and being hostile toward the Quarg means never meeting Elias Hanover, so alternative paths for Quarg-hostile players would obviously require a lot of work to fit, but in the end I think it would be worth it.

Another, simpler fix would be making it so that you can somehow make the Quarg friendly, although what they would entail I do not know.

**Comment by Bladewood** [comment]
Money is no issue.

**Comment by KyroFirehammer** [comment]
And posdobly a bounty mission on a hostile government

**Comment by gunqqer** [comment]
Currently 50-100M would be a cap or two of a wardragon (or what ever one is cappable) or 2 raiders. 100m isn't much late game (So durring the wanderer story line). Buying rep isn't really something many repectable people would do (so buying pirate rep would be fine.) Also the Quarg are not going to ever have bounty missions, it isn't their nature

**Comment by ghoulavenger** [comment]
One of the reasons why I abandoned a playthrough where I had plundered quarg tech, was because of finding out about missions involving the quarg. So I would welcome changes to the wanderer questline BUT the problem is that the quarg tech kind of breaks the game -- most specifically the skylance, but the batteries are pretty useful too. So if you were going to do it, you would first need to rebalance the quarg imho.

As far as money concerns go, because of depreciation 2 raiders is only worth about 25 mil and you can only really capture one at a time before having to get new crew anyway. So this 50-100 mil in a capture or two is slightly off. But, I make about 25-30 million credits a jump when I'm doing trading (I have 100 deep rivers and could easily expand my fleet for more), so it is still pretty cheap for me. But if you're doing wanderer missions, guess what? Their ships cost 70+ mil anyway. So a 50 mil reparation is less than buying a warship. If you were trying to push reparations, you'd need to push it to cover at least the value of the warships before depreciation, which should probably be 100 mil per ship killed, give or take. So if you're actually killing Quarg, buying their friendship shouldn't be easy, and should definitely not reward killing more Quarg.

**Comment by Lineth** [comment]
Yeah no amount of money should be able to get in the Quarg's good graces for reasons already stated. Even missions are eh. _Maybe_ there can be a 'last chance' mission to redeem yourself if your reputation with them is not too low. And even that would require you jumping through hoops or siding against the Coalition or something.

**Comment by Bladewood** [comment]
@Lineth I agree with that whole first paragraph, though we may have to consider the possible situation where the player may not yet have met the Coalition and the Heliarchs.

This is my position on the second:
At the moment, the Quarg have no need for ship or weapon diversity because everything they have just *works*.
That, of course, could all change once they have to go up against the Coalition again, because they are much stronger than before (and generally stronger than most races the Quarg may face).

**Comment by gunqqer** [comment]
Yeah having a Coalition vs Quarg storyline would be nice. Maybe... nope, save it for a different issue, gunqqer. I agree with bladewood, the quarg don't need new stuff until we get new story. As he said, it works. It really does.

**Comment by KyroFirehammer** [comment]
What i meant by bounty was a VERY hard mission that would "prove" tour loyalty to the ( this could also work with the republic , hai and other english( ?) Speaking people's (possibly wanderers wotgthe communicator too)

**Comment by tehhowch** [comment]
Attacking someone, stealing their tech, and then hunting their enemies to regain favor doesn't sound very loyal. It sounds very "freelance" and I expect the Quarg to see right through any such schemes

**Comment by KyroFirehammer** [comment]
Perhaps but i was thinkingvalongvthe likrs of a HUGE sestor (or coalition but idk how) where the quarg are trapped on a deserted planet and cannot leave because they are out of hyperspace fuel. You then destroy all of the sestor ships ,land on the planet refuell the ships and escort them bach to the ringworld. This wouldn't completely fix your rep if you've killed several ships but if you haven't destroyed any it would more or less bring you onto good terms with them. (If you have capped many of their ships/blew them up it would require like 3 or 4 of these ridiculously hard fights/ other missions and possibly a large heavy fine)

**Comment by tehhowch** [comment]
T3+ civilizations do not "run out of hyperspace fuel". 
```
ship "Quarg Skylark"
	sprite "ship/skylark"
	attributes
		"ramscoop" 10

ship "Quarg Wardragon"
	sprite "ship/wardragon"
	attributes
		"ramscoop" 10
```

The problem with advanced civilizations (IMO especially ones lored like the Quarg) is that they **understand your motivations/intentions** and thus *should not trust you* if you have ever willingly attacked them and been successful. To add a "rep fix" that treats the player as more advanced or more capable than them is logically incompatible, especiallywhen there are 1 **trillion** Quarg per ringworld. 
To me the only plausible "rep fix" missions are where `<alien species>` asks the player to do something that is needed for the aliens, yet incompatible with the aliens' social / technological constructs - e.g. undercover / illegal work. And yet then again the player is demonstrating a willingness to act in an unsavory manner, which leads back to the bit about the player being an untrustworthy character.

For aliens which are not lored as the Quarg, sure, "bounty hunting" or "underhanded change agent" rep fix missions may be plausible. For the Quarg? I'm not sure it **ever** makes sense for your reputation to be repairable.

**Comment by Pointedstick** [comment]
The obvious solution is to make Quarg ships uncappable and outfits unplunderable by the player until these sorts of issues are worked out.

**Comment by gunqqer** [comment]
But making quarg stuff uncappable and unplunderable will piss a lot of people off, myself included.

**Comment by Pointedstick** [comment]
Why, out of curiosity?

**Comment by Lineth** [comment]
Because cheating? I mean, I personally consider Quarg stuff to be as good as a cheat considering it's pretty much all placeholders and will likely get depreciated once the Quarg revamp hits.

**Comment by Pointedstick** [comment]
> Because cheating?

Yeah, that's sort of what I was getting at. Preserving people's ability to break the game for their own amusement has to be a pretty low priority compared to storyline integrity.

**Comment by KyroFirehammer** [comment]
Or just update te quarg (and coalition) to where they are on level with eachother (nerf quarg and SLIGHTLY buff coalirion [add a better coalition weapon]) and possibly have the two fight against eachother in short skirmishes ( storyline mabie *wonk wink)

**Comment by MessyMix** [comment]
they're already on a very similar level. Not a fan of this idea. (your other one was better)

and, the quarg are going to have to be redone anyways. Although, tbh, if the Punisher heavy warship stays where it's at statistically, and the Wardragon is reclassed as a light warship as the description suggests, then there's no way the Coalition stood a chance against the quarg. Given the strength of the Arfecta, once the Wardragon gets reclassed as a light warship (please not a Medium or Heavy), the tougher quarg ships would be impossibly strong compared to the Coalition. does this justify a heliarch buff? No way could it have stood up to Quarg mediums / heavies.

**Comment by KiLEdEnNis** [comment]
the Coalition back in its day could have greatly outnumbered the Quarg ship-wise though, that's how they could have stood up against the Quarg
assuming the Quarg really fought back fiercely and didn't give up that area of space because they don't feel like massacring 3 species to stay in an area of space where they are unwanted

**Comment by MessyMix** [comment]
Lol, can you imagine trying to swarm a single Quarg Heavy Warship using saryd sojourners... 🤣

**Comment by Lineth** [comment]
I'm imagining the Wardragon would be completely reworked or replaced. It's a statistical mess. Probably make it a medium and replace all it's sprites for ones that match the Quarg's aesthetics.

**Comment by KyroFirehammer** [comment]
They would have to have a weapon on par or better than a skylance to even come close to beating quarg. The whole coalition being able to kill quarg seems unlikely, it would make more sense if along the way you find out the out or another alien race bad helped them ( this would also explain why they have only 1 remaining system after you take deneb) but keep it hushed up to keep their citizens in line through fear and would further deepen the plot and reasoning on the weapon ban in coalition space.

**Comment by jafdy** [comment]
@KyroFirehammer Do you make this up as you go along?

OK the Pug might have helped the coalition but it would have been the coalition that did the fighting. 
The player doesn't need skylance level weaponry to take on the Quarg, why should the coalition.
Most likely the Quarg left because they didn't want to be at constant war.

**Comment by gunqqer** [comment]
That's the thing about war: you don't need to destroy the enemey, you need to just push them until they give up. For the Quarg they don't like war (Full out war, them defending the Kor Efrit from the Automatons isn't war) and don't want to be engaged in a war. They still want to defend their ringworlds, but there's a point where their losses arn't worth keeping the ringworld

**Comment by KyroFirehammer** [comment]
Yes but the player also needs dozens of 349s and 512s to even think about it along with wanderer ad korath weapons combined. There is no way a bunch of transports and freighters could even think about it it's unrealistic to even think they could, publishers might ,but they aren't made till after the overthrow. The only possible and logical way they could is if the pug helped them similarly to the deep or the wanderers. Pug  because they are the only ones remotely close to them other than quarg space.

**Comment by gunqqer** [comment]
I can take out a quarg with a few shield beetles and some flame throwers. (Plus something for damage). My fleet of 10 349s with 6 skylances on each are total overkill for quarg

**Comment by KyroFirehammer** [comment]
Yeah but still A shield Beatle are way better than a freighter and even then that's 1 singled out quarg not a large fleet of them

**Comment by gunqqer** [comment]
So are you sugesting I get a fleet of frieghters to take on a few quarg... I almost want to accept that challenge

**Comment by ghoulavenger** [comment]
Eh, why not, I've seen youtube vids of people doing it with star barges.

**Comment by endlesssteve** [comment]
Back to the original point.
I think it's much more in keeping with the story that the Quarg choose to view the big picture and temporarily turn neutral to the player. May be some extra diplomatic mission to arrange the "cease fire".

**Comment by KyroFirehammer** [comment]
Haha lol I'm looking up star barge vs quarg

**Comment by Pointedstick** [comment]
> Haha lol I'm looking up star barge vs quarg

This is not a forum! Y'all need to realize that the **more irrelevant chatter there is in a bug report, the less likely it is to be fixed.**

**Comment by FixItYondu** [comment]
Is there any chance of this getting feature being added to the next update, or is it going to be added to 1.0.0 whenever that is released?

**Comment by Amazinite** [comment]
Before this can be done, I'd say that the Wanderer story needs to first be finished so that we can properly assess the impact that any alternative routes might have on the rest of the story. Wouldn't want to create an alternative route that then doesn't make sense with the plot of the end of the campaign. As for when the Wanderer story gets finished, that's unknown. @endless-sky is the one working on it, but the last update we got on that was more than two years ago with the addition of Tele'ekite ships and weapons (https://github.com/endless-sky/endless-sky/commit/c6c5d149262e74662c0c82f076af35f130dfdfbc, https://github.com/endless-sky/endless-sky/commit/1b8ecea63da7701c19f1c3db057a34554e83d7f9) and mention from MZ that he's had difficulties figuring out how the Wanderer campaign should even end.
